install/kubernetes/istio-auth-with-cluster-ca.yaml

```diff
@@ -384,10 +384,11 @@ data:
     # Uncomment the following line to enable mutual TLS between proxies
     authPolicy: MUTUAL_TLS
     mixerAddress: istio-mixer:9091
-    discoveryAddress: istio-pilot:8080
```

Comment:
shouldn't you keep that for backward compatibility for now (until we get a new pilot)